src/Microsoft.FeatureManagement/FeatureManager.cs

```diff
@@ -225,10 +225,10 @@ public async ValueTask<Variant> GetVariantAsync(string feature, CancellationToke
         /// Gets the assigned variant for a specific feature.
         /// </summary>
         /// <param name="feature">The name of the feature to evaluate.</param>
-        /// <param name="context">An instance of <see cref="TargetingContext"/> used to evaluate which variant the user will be assigned.</param>
+        /// <param name="context">A context providing information that can used to evaluate which variant the user will be assigned.</param>
```

Comment:
A context providing information that can be used to evaluate which variant will be assigned to the user.

Reply:
I followed the existing [comments](https://github.com/microsoft/FeatureManagement-Dotnet/blob/preview/src/Microsoft.FeatureManagement/IVariantFeatureManager.cs#L35).

Reply:
I’m not sure which comments you’re referring to, but the current one you have contains grammatical errors.

Reply:
Line 35 in IVariantFeatureManager.cs. 

> but the current one you have contains grammatical errors.

Ok, I will change all of them.